### DISCUSSION TITLE
How to load SVG file with picture embedded in image tag?

### DISCUSSION THREAD

**Discussion by bchrescionko** [thorvg/thorvg#1305] [Q&A] [answered]
Hello,

I tried to load my svg file with one of example file named src/examples/Svg2.cpp. Only thing I'm getting is empty, white window. I checked with other svg file that worked and difference between them is that my contains image tag. I tried with other svg file that contain this tag and got the same result, empty, white window. 

In example file Svg2.cpp I only change line 44 so now I pass path to my svg file to picture->load. My question is how can I display svg file with <image> tag with the help of thorvg.

**Comment by JSUYA** [comment]
Could you please share SVG file it is? And please tell me the exact tag name. 
And if it's a bug, I think the issue page is more appropriate :)

**Reply by bchrescionko** [reply]
![test_bluedot](https://user-images.githubusercontent.com/91251249/212089170-d92bff46-f9da-42e7-9508-ce16ff532a76.svg)
 The name of tag is image:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image

To be honest I don't know if it is a bug. I can just load this file wrong way.

**Comment by mgrudzinska** [comment, marked as answer]
@bchrescionko I'm just guessing what may be your problem - do you have an embedded png/jpg inside your svg file? if so, try to enable the png/jpg loaders.

**Reply by bchrescionko** [reply]
Yes u were right, I just needed to enable png loder in meson_options
